### PR TITLE
Update Mesos version to 0.28.0

### PR DESCRIPTION
**Pull request by rveznaver**

### CODE REVIEW

**Comment by FlorentFlament** [comment]
+1

File: .kitchen.ec2.yml [hunk 1]
```diff
@@ -43,67 +43,50 @@ platforms:
       - recipe[curl]
     driver:
       image_id: ami-2a4d1a40
-    transport:
```
Comment (FlorentFlament) [inline .kitchen.ec2.yml]: For my information, what is `transport` used for ?

Comment (rveznaver) [inline .kitchen.ec2.yml]: We have used it to specify the username for the given AMI on EC2. Apparently, now the kitchen-ec2 gem selects the proper username automatically.

**Comment by pierrecdn** [comment]
OPT: 0.28.1 is the current latest.

**Comment by rveznaver** [comment]
@pierrecdn Can't find it on http://open.mesosphere.com/downloads/mesos/

**Comment by pierrecdn** [comment]
OK. In practice, the packages for this version are available in the different repos.
Let's wait for an update of the webpage.

**Pull request merged**